### ISSUE TITLE
update from version 2.2.0.10 to 2.2.0.11 failed (OS Version '10.0.15063.0' not supported?) (Solved! see Post #987)

### ISSUE THREAD

**Issue by Grumpelumpf** [github/VisualStudio#1132]
- GitHub Extension for Visual Studio  version: 2.2.0.10/2.2.0.11
- Visual Studio version: Community 2017 15.2 (26430.16)

```
10.08.2017 17:21:29 - Microsoft VSIX Installer
10.08.2017 17:21:29 - -------------------------------------------
10.08.2017 17:21:29 - vsixinstaller.exe version:
10.08.2017 17:21:29 - 15.0.26430.16 built by: D15SVC
10.08.2017 17:21:29 - -------------------------------------------
10.08.2017 17:21:29 - Command line parameters:
10.08.2017 17:21:29 - C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\vsixinstaller.exe,/appidinstallpath:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe,/skuName:Community,/skuVersion:15.0.26430.16,/appidname:Microsoft Visual Studio Community 2017,T:\Temp\VSIX1tzbvk0v.vsix,/installas:2320,/callingprocessid:1996
10.08.2017 17:21:29 - -------------------------------------------
10.08.2017 17:21:29 - Microsoft VSIX Installer
10.08.2017 17:21:29 - -------------------------------------------
10.08.2017 17:21:29 - Initializing Install...
10.08.2017 17:21:30 - Extensiondetails...
10.08.2017 17:21:30 - 	Identifier         : c3d3dc68-c977-411f-b3e8-03b0dccf7dfc
10.08.2017 17:21:30 - 	Name               : GitHub Extension for Visual Studio
10.08.2017 17:21:30 - 	Author             : GitHub, Inc
10.08.2017 17:21:30 - 	Version            : 2.2.0.11
10.08.2017 17:21:30 - 	Description        : A Visual Studio Extension that brings the GitHub Flow into Visual Studio.
10.08.2017 17:21:30 - 	Locale             : en-US
10.08.2017 17:21:30 - 	MoreInfoURL        : https://visualstudio.github.com/
10.08.2017 17:21:30 - 	InstalledByMSI     : False
10.08.2017 17:21:30 - 	SupportedFrameworkVersionRange : [4.5,)
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - 	SignatureState     : ValidSignature
10.08.2017 17:21:30 - 	SignedBy           : GitHub, Inc.
10.08.2017 17:21:30 - 	Certificate Info   : 
10.08.2017 17:21:30 - 		-------------------------------------------------------
10.08.2017 17:21:30 - 		[Subject]       : CN="GitHub, Inc.", O="GitHub, Inc.", L=San Francisco, S=California, C=US
10.08.2017 17:21:30 - 		[Issuer]        : CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
10.08.2017 17:21:30 - 		[Serial Number] : 013AA7BBB950DCCD25305F1602258B00
10.08.2017 17:21:30 - 		[Not Before]    : 23.11.2016 01:00:00
10.08.2017 17:21:30 - 		[Not After]     : 19.11.2019 13:00:00
10.08.2017 17:21:30 - 		[Thumbprint]    : C4FDF13659F49E57AC14774FDF45053F499F185F
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - 	Unterstützte Produkte : 
10.08.2017 17:21:30 - 		Microsoft.VisualStudio.Community
10.08.2017 17:21:30 - 			Version : [14.0,15.0]
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - 	Verweise           : 
10.08.2017 17:21:30 - 		-------------------------------------------------------
10.08.2017 17:21:30 - 		Identifier   : Microsoft.VisualStudio.MPF.14.0
10.08.2017 17:21:30 - 		Name         : Visual Studio MPF 14.0
10.08.2017 17:21:30 - 		Version      : [14.0,)
10.08.2017 17:21:30 - 		MoreInfoURL  : 
10.08.2017 17:21:30 - 		Nested       : No
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - 	Voraussetzungen    : 
10.08.2017 17:21:30 - 		-------------------------------------------------------
10.08.2017 17:21:30 - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
10.08.2017 17:21:30 - 		Name         : Visual Studio core editor
10.08.2017 17:21:30 - 		Version      : [15.0.25824.0,16.0)
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - Signaturdetails...
10.08.2017 17:21:30 - 	Extension is signed with a valid signature.
10.08.2017 17:21:30 - 
10.08.2017 17:21:30 - PKGDEF Information: Looking for master PkgDef file, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\master.pkgdef

10.08.2017 17:21:30 - PKGDEF Information: Creating PkgDefCacheNonVolatile
10.08.2017 17:21:30 - PKGDEF Information: Double-checking master pkgdef file
10.08.2017 17:21:30 - PKGDEF Information: PkgDefManagement initialized
10.08.2017 17:21:30 - PKGDEF Information: RootFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\

10.08.2017 17:21:30 - PKGDEF Information: ShellFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\

10.08.2017 17:21:30 - PKGDEF Information: PkgDefSearchPath, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.admin.pkgdef

10.08.2017 17:21:30 - PKGDEF Information: ImageManifestSearchPath, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions;C:\Users\Ich\AppData\Local\Microsoft\VisualStudio\15.0_fda4cc54\Extensions

10.08.2017 17:21:30 - PKGDEF Information: ApplicationExtensionsFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions

10.08.2017 17:21:30 - PKGDEF Information: "ZeroImpact" = dword:0, "MergeRegistry" = dword:3
10.08.2017 17:21:30 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
10.08.2017 17:21:30 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
10.08.2017 17:21:30 - PKGDEF Information: User extensions enabled by setting, Path: HKEY_CURRENT_USER\Software\Microsoft\VisualStudio\15.0_fda4cc54\ExtensionManager\EnableAdminExtensions

10.08.2017 17:21:30 - PKGDEF Information: PkgDefCache flags, PKGDEF: 7001
10.08.2017 17:21:30 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
10.08.2017 17:21:30 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
10.08.2017 17:21:30 - PKGDEF Information: PkgDef cache is current.
10.08.2017 17:21:30 - PKGDEF Information: PkgDefManagement startup complete
10.08.2017 17:21:31 - Für die Extension wird ein Upgrade von Version 2.2.0.10 durchgeführt.
10.08.2017 17:22:37 - Folgende Zielprodukte wurden ausgewählt...
10.08.2017 17:22:37 - 	Microsoft Visual Studio Community 2017
10.08.2017 17:22:37 - 
10.08.2017 17:22:38 - Die Installation der Extension für Microsoft Visual Studio Community 2017 wurde gestartet...
10.08.2017 17:22:38 - "GitHub Extension for Visual Studio" wird aktualisiert, Version 2.2.0.10 auf Version 2.2.0.11.
10.08.2017 17:22:38 - VSIX TaskScheduler: VSIX Auto Updatetask ist bereits vorhanden.
10.08.2017 17:22:41 - Package Microsoft.Windows.UniversalCRT.Msu.8 is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.2,6.3)'.
10.08.2017 17:22:41 - Package Microsoft.Windows.UniversalCRT.Msu.81 is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.3,6.4)'.
10.08.2017 17:22:41 - Package Microsoft.Net.4.6.FullRedist is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1.1,6.4)'.
10.08.2017 17:22:41 - Package Microsoft.Net.4.6.FullRedist.Resources is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1.1,6.4)'.
10.08.2017 17:22:41 - Package Microsoft.Windows.UniversalCRT.Msu.7 is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.2)'.
10.08.2017 17:22:41 - Package Microsoft.DiagnosticsHub.KB2882822.Win7 is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.2)'.
10.08.2017 17:22:41 - Package WebSocket4NetV2 is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.2)'.
10.08.2017 17:22:41 - Package Microsoft.Net.4.6.1.FullRedist is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1.1,10.0.10241)'.
10.08.2017 17:22:41 - Package Microsoft.Net.4.6.1.FullRedist.Resources is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1.1,10.0.10241)'.
10.08.2017 17:22:41 - Package Microsoft.Windows.81SDK.Store.DirectX.Msi is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.3]'.
10.08.2017 17:22:41 - Package Microsoft.Windows.81SDK.Store.DirectX.Msi is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.3]'.
10.08.2017 17:22:41 - Package Microsoft.Windows.81SDK.Desktop.DirectX.Msi is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.3]'.
10.08.2017 17:22:41 - Package Microsoft.Windows.81SDK.Desktop.DirectX.Msi is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '[6.1,6.3]'.
10.08.2017 17:22:41 - Package Win10_Emulator_10.0.15063 is not applicable. Skipping it due to the following reason(s): Hyper-V is not supported by the current environment.
10.08.2017 17:22:41 - Package Microsoft.VisualStudio.OfficeDeveloperTools.WindowsIdentityFoundation.Msu is not applicable. Skipping it due to the following reason(s): The current OS Version '10.0.15063.0' is not in the supported version range '(,6.2)'.
10.08.2017 17:22:41 - Package Win10_Emulator_10.0.14393.0 is not applicable. Skipping it due to the following reason(s): Hyper-V is not supported by the current environment.
10.08.2017 17:22:41 - Package vs.emulator.android.bundle is not applicable. Skipping it due to the following reason(s): Hyper-V is not supported by the current environment.
10.08.2017 17:22:41 - Package vs.emulator.android is not applicable. Skipping it due to the following reason(s): Hyper-V is not supported by the current environment.
10.08.2017 17:22:47 - Package 'GitHub.VisualStudio,version=2.2.0.11' failed to install. System.IO.IOException: The file 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions\dphzvrzq.gnr\Akavache.dll' already exists.
   bei Microsoft.VisualStudio.Setup.Installer.VsixInstaller.InstallCore(String localPath, String installDir, InstallablePackage pkg)
   bei Microsoft.VisualStudio.Setup.Installer.InstallerBase.InstallHelper(InstallData installData)
10.08.2017 17:22:47 - Skipping cache of package 'Component.GitHub.VisualStudio,version=2.2.0.11' since the package or a parent package had a vital failure.
10.08.2017 17:22:53 - Installationsfehler: Microsoft.VisualStudio.Setup.PackageFailureException: Package 'GitHub.VisualStudio' failed to install
   bei Microsoft.VisualStudio.Setup.InstallOperation.Run(CancellationToken token)
   bei Microsoft.VisualStudio.Setup.Engine.RunOperation(InstallOperation installOperation, CancellationToken token, ExecuteAction action, ITelemetryOperation telemetryOperation)
   bei Microsoft.VisualStudio.Setup.Engine.RunCoreOperation(InstallOperation coreOperation, ExecuteAction action, ITelemetryOperation telemetryOperation, CancellationToken token)
   bei Microsoft.VisualStudio.Setup.Engine.Install(Product product, String destination, CancellationToken token)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.PerformSetupEngineInstall(InstallableExtensionImpl extension, Boolean installPerMachine, Boolean isPackComponent, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, IProgress`1 progress, InstallFlags installFlags, AsyncOperation asyncOp, Version targetVsVersion, IInstalledExtension& newExtension)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallInternal(InstallableExtensionImpl extension, InstallFlags installFlags, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, IProgress`1 progress, Version targetVsVersion)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.BeginInstall(IInstallableExtension installableExtension, InstallFlags installFlags, AsyncOperation asyncOp, Version targetVsVersion)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallWorker(IInstallableExtension extension, InstallFlags installFlags, AsyncOperation asyncOp)
```

**Comment by jcansdale** [comment]
Hi @Grumpelumpf,

I think it is likely that this is the critical line:
```
10.08.2017 17:22:47 - Package 'GitHub.VisualStudio,version=2.2.0.11' failed to install. System.IO.IOException: The file 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions\dphzvrzq.gnr\Akavache.dll' already exists.
```

Could you try going into `Extensions and Updates` and Uninstall the GitHub Extension for Visual Studio.
![image](https://user-images.githubusercontent.com/11719160/29413232-94052fe0-8353-11e7-8b54-52b3f68da534.png)

If this works, could you try installing again. If the uninstall fails, could you send me the log?

Thanks.

**Comment by Grumpelumpf** [comment]
Hi @jcansdale ,

tried to uninstall and get this error protokoll:
17.08.2017 16:49:58 - Microsoft VSIX Installer
17.08.2017 16:49:58 - -------------------------------------------
17.08.2017 16:49:58 - vsixinstaller.exe version:
17.08.2017 16:49:58 - 15.0.26430.16 built by: D15SVC
17.08.2017 16:49:58 - -------------------------------------------
17.08.2017 16:49:58 - Command line parameters:
17.08.2017 16:49:58 - C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\vsixinstaller.exe,/appidinstallpath:C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe,/skuName:Community,/skuVersion:15.0.26430.16,/appidname:Microsoft Visual Studio Community 2017,/u:c3d3dc68-c977-411f-b3e8-03b0dccf7dfc,/installas:1404,/callingprocessid:23192
17.08.2017 16:49:58 - -------------------------------------------
17.08.2017 16:49:58 - Microsoft VSIX Installer
17.08.2017 16:49:58 - -------------------------------------------
17.08.2017 16:49:59 - PKGDEF Information: Looking for master PkgDef file, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\master.pkgdef

17.08.2017 16:49:59 - PKGDEF Information: Creating PkgDefCacheNonVolatile
17.08.2017 16:49:59 - PKGDEF Information: Double-checking master pkgdef file
17.08.2017 16:49:59 - PKGDEF Information: PkgDefManagement initialized
17.08.2017 16:49:59 - PKGDEF Information: RootFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\

17.08.2017 16:49:59 - PKGDEF Information: ShellFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\

17.08.2017 16:49:59 - PKGDEF Information: PkgDefSearchPath, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.admin.pkgdef

17.08.2017 16:49:59 - PKGDEF Information: ImageManifestSearchPath, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions;C:\Users\Ich\AppData\Local\Microsoft\VisualStudio\15.0_fda4cc54\Extensions

17.08.2017 16:49:59 - PKGDEF Information: ApplicationExtensionsFolder, Path: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\Extensions

17.08.2017 16:49:59 - PKGDEF Information: "ZeroImpact" = dword:0, "MergeRegistry" = dword:3
17.08.2017 16:49:59 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
17.08.2017 16:49:59 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
17.08.2017 16:49:59 - PKGDEF Information: User extensions enabled by setting, Path: HKEY_CURRENT_USER\Software\Microsoft\VisualStudio\15.0_fda4cc54\ExtensionManager\EnableAdminExtensions

17.08.2017 16:49:59 - PKGDEF Information: PkgDefCache flags, PKGDEF: 7001
17.08.2017 16:49:59 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
17.08.2017 16:49:59 - PKGDEF Information: Could not find ConfigurationChanged timestamp., PKGDEF: 80070002
17.08.2017 16:49:59 - PKGDEF Information: PkgDef cache is current.
17.08.2017 16:49:59 - PKGDEF Information: PkgDefManagement startup complete
17.08.2017 16:49:59 - Initializing Uninstall...
17.08.2017 16:49:59 - Extensiondetails...
17.08.2017 16:49:59 - 	Identifier         : c3d3dc68-c977-411f-b3e8-03b0dccf7dfc
17.08.2017 16:49:59 - 	Name               : GitHub Extension for Visual Studio
17.08.2017 16:49:59 - 	Author             : GitHub, Inc
17.08.2017 16:49:59 - 	Version            : 2.2.0.10
17.08.2017 16:49:59 - 	Description        : A Visual Studio Extension that brings the GitHub Flow into Visual Studio.
17.08.2017 16:49:59 - 	Locale             : en-US
17.08.2017 16:49:59 - 	MoreInfoURL        : https://visualstudio.github.com/
17.08.2017 16:49:59 - 	InstalledByMSI     : False
17.08.2017 16:49:59 - 	SupportedFrameworkVersionRange : [4.5,)
17.08.2017 16:49:59 - 
17.08.2017 16:49:59 - 	Unterstützte Produkte : 
17.08.2017 16:49:59 - 		Microsoft.VisualStudio.Community
17.08.2017 16:49:59 - 			Version : [14.0,15.0]
17.08.2017 16:49:59 - 
17.08.2017 16:49:59 - 	Verweise           : 
17.08.2017 16:49:59 - 		-------------------------------------------------------
17.08.2017 16:49:59 - 		Identifier   : Microsoft.VisualStudio.MPF.14.0
17.08.2017 16:49:59 - 		Name         : Visual Studio MPF 14.0
17.08.2017 16:49:59 - 		Version      : [14.0,)
17.08.2017 16:49:59 - 		MoreInfoURL  : 
17.08.2017 16:49:59 - 		Nested       : No
17.08.2017 16:49:59 - 
17.08.2017 16:49:59 - 	Voraussetzungen    : 
17.08.2017 16:49:59 - 		-------------------------------------------------------
17.08.2017 16:49:59 - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
17.08.2017 16:49:59 - 		Name         : Visual Studio core editor
17.08.2017 16:49:59 - 		Version      : [15.0.25824.0,16.0)
17.08.2017 16:49:59 - 
17.08.2017 16:49:59 - 
17.08.2017 16:50:26 - Folgende Zielprodukte wurden ausgewählt...
17.08.2017 16:50:26 - 	Microsoft Visual Studio Community 2017
17.08.2017 16:50:26 - 
17.08.2017 16:50:27 - GitHub Extension for Visual Studio Version 2.2.0.10 wird deinstalliert.
17.08.2017 16:50:31 - System.InvalidOperationException: Zum Installieren/Deinstallieren ist "Component" erforderlich, jedoch nicht im VSIX-Katalog enthalten.
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.PerformSetupEngineUnInstall(IInstalledExtension extension)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.UninstallInternal(IInstalledExtension extension, Boolean forceDelete, Version targetedVSVersion)
   bei Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.Uninstall(IInstalledExtension extension, Version targetedVSVersion)
   bei VSIXInstaller.InstallProgressPage.UninstallVsix(String vsixId, SupportedVSSKU targetSku, IProgress`1 progress)

**Comment by jcansdale** [comment]
I'm getting something similar in one of my instances of Visual Studio 2017 (I have several installed).
```
17/08/2017 16:21:40 - The following target products have been selected...
17/08/2017 16:21:40 - 	Microsoft Visual Studio Community 2017
17/08/2017 16:21:40 - 
17/08/2017 16:21:41 - Uninstalling 'GitHub Extension for Visual Studio', version 2.2.0.10.
17/08/2017 16:21:46 - System.InvalidOperationException: The VSIX's catalog does not include a 'Component' which is required for install/uninstall.
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.PerformSetupEngineUnInstall(IInstalledExtension extension)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.UninstallInternal(IInstalledExtension extension, Boolean forceDelete, Version targetedVSVersion)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.Uninstall(IInstalledExtension extension, Version targetedVSVersion)
   at VSIXInstaller.InstallProgressPage.UninstallVsix(String vsixId, SupportedVSSKU targetSku, IProgress`1 progress)
```

@Grumpelumpf, have you also installed multiple instances of VS 2017?

**Comment by Grumpelumpf** [comment]
No ... only an "old" VS 2013 (?) installation, but uninstalled several days ago.

**Comment by grokys** [comment]
This looks like a related issue: https://github.com/github/VisualStudio/issues/987

**Comment by jcansdale** [comment]
I think it is related.

On my broken instance, I followed @tinaschrepfer's advice:

> I would also add that you should remove the GitHub extension for Visual Studio from VS Installer prior to removing the DLLs to make sure you don't corrupt the setup state. To do so, go to VS Installer, and then the Individual Component tab, unselect "GitHub extension for Visual Studio", and then click Modify.

Unfortunately I now get the following when I try to reinstall.

```
17/08/2017 16:54:53 - Skipping 'C:\Program Files (x86)\Common Files\Microsoft\ExtensionManager\Extensions' as it doesn't exist
17/08/2017 16:54:53 - Skipping 'C:\Users\passp\AppData\Local\Microsoft SDKs' as it doesn't exist
17/08/2017 16:54:53 - An extension with a matching Identifier is already installed to this product.
17/08/2017 16:54:54 - Microsoft.VisualStudio.ExtensionManager.CorruptInstanceException: VSIX Installer has encountered a problem. To troubleshoot, follow the steps here: https://aka.ms/pc5ifb ---> Microsoft.VisualStudio.Setup.Dependencies.DependencyGraphConstructionException: The dependent package cannot be found: GitHub.VisualStudio,version=2.3.0.18
   at Microsoft.VisualStudio.Setup.Dependencies.DependencyManager.FindDependentPackages(Dependency dependency, DependencyComparer comparer, Dictionary`2 packageIndex)
   at Microsoft.VisualStudio.Setup.Dependencies.DependencyManager.BuildGraphRecursive(IPackage graphRoot, DependencyNode node, DependencyComparer comparer, Dictionary`2 packageIndex, Dictionary`2 nodeIndex, Boolean splitNodes)
   at Microsoft.VisualStudio.Setup.Dependencies.DependencyManager.BuildGraphRecursive(IPackage graphRoot, DependencyNode node, DependencyComparer comparer, Dictionary`2 packageIndex, Dictionary`2 nodeIndex, Boolean splitNodes)
   at Microsoft.VisualStudio.Setup.Dependencies.DependencyManager.BuildGraphCore(IPackage root, IEnumerable`1 packages, DependencyComparer comparer, Dictionary`2 packageIndex, Dictionary`2 nodeIndex, Boolean splitNodes, DependencyCollection deprecated)
   at Microsoft.VisualStudio.Setup.Dependencies.DependencyManager.BuildGraphCandidatePackages(IPackage root, IEnumerable`1 packages, Boolean isUpdate, DependencyCollection deprecated)
```

@tinaschrepfer do you have any advice on what to try next? I'd rather not resort to repaving all instances of VS 2017 if at all possible. Thanks for any suggestions.

**Comment by Grumpelumpf** [comment]
solved with this solution: https://github.com/github/VisualStudio/issues/987#issuecomment-300755269

**Comment by jcansdale** [comment]
Glad to hear you managed to fix it!  Is `2.2.0.11` installed now?

**Comment by Grumpelumpf** [comment]
Yes, 2.2.0.11 installed without any issue. thx
